### PR TITLE
Adding MVVM friendly sample 

### PR DESCRIPTION
**Pull request by wm-v-michaels**
Adding MVVM friendly sample using ViewModel and .net behavior to handle player creation and reinitialization.

### CODE REVIEW

File: src/Samples/Samples.Wpf.MVVM/ViewModels/MainViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+using GalaSoft.MvvmLight; /* Or your MVV provider of choice */
```
Comment (jeremyVignelles) [inline src/Samples/Samples.Wpf.MVVM/ViewModels/MainViewModel.cs]: I'd prefer a sample that builds out of the box, without installing a MVVM provider. Would that be possible to write a INotifyPropertyChanged implementation directly instead?

**Comment by wm-v-michaels** [comment]
I’ll give it a try tomorrow. I wrote it into my code using mvvmlight, and kind of abstracted it for this sample, so it’ll take me a little longer to write it with vanilla notifypropertychanged stuff.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by wm-v-michaels** [comment]
I've added a basic ViewModelBase to the project, and commented out references to MVVMlight.  

I'm not certain if it would work in Windows Forms, but if winforms supports raising events like WPF, it might be easily adaptable.  Not having worked in winforms in a while, I can't really answer that question offhand.

File: src/Samples/Samples.Wpf.MVVM/ViewModels/MainViewModel.cs (hunk 1 above)
Comment (wm-v-michaels) [reply src/Samples/Samples.Wpf.MVVM/ViewModels/MainViewModel.cs]: Does it work for you now?

**Comment by jeremyVignelles** [comment]
Hi, sorry for the delay,

While this is a good PR, I don't think that I will merge it because:

- Only the source is available through data binding. Real-world examples would probably expect Time, position etc... as it was suggested by #246
- It uses the Wpf project, which should not be the default choice, even when developing with Wpf (my bad for the confusion !)
- It can still serve as an example for those who ask google "How to use Vlc.DotNet with MVVM"
- It does not fix any bug, the effort is not worth it, as I tend to phase out this project in favor of LibVLCSharp.

Thanks anyway for your contribution and the time you spent fixing this !

**Pull request closed without merging**